src/concrete/ml/quantization/quantized_module.py

```diff
@@ -142,6 +146,14 @@ def __init__(
         # Input-output quantizer mapping for composition is not enabled at initialization
         self._composition_mapping: Optional[Dict] = None
 
+        # Initialize _preprocessing_numpy
```

Comment:
small bad refactor artifact : it should be `_preprocessing_module`

Reply:
thanks!